shared/package-lock.json

```diff
@@ -162,9 +162,9 @@
       }
     },
     "eslint-config-prettier": {
-      "version": "8.3.0",
-      "resolved": "https://registry.npmjs.org/eslint-config-prettier/-/eslint-config-prettier-8.3.0.tgz",
-      "integrity": "sha512-BgZuLUSeKzvlL/VUjx/Yb787VQ26RU3gGjA3iiFvdsp/2bMfVIWUVP7tjxtjS0e+HP409cPlPvNkQloz8C91ew==",
+      "version": "8.5.0",
```

Comment:
Why did `shared/package-lock.json` change without `shared/package.json` changing?

Reply:
as discussed, reflects current state in `release-al2` which is inconsistent